core/src/androidTest/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/YouTubePlayerViewClickTest.kt

```diff
@@ -0,0 +1,130 @@
+package com.pierfrancescosoffritti.androidyoutubeplayer.core
+
+import android.app.Activity
+import android.view.MotionEvent
+import android.view.View
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.testActivity.TestActivity
+import com.pierfrancescosoffritti.androidyoutubeplayer.test.R
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.verify
+import java.util.*
+
+@LargeTest
+class YouTubePlayerViewClickTest {
+
+    @Rule
+    @JvmField
+    val activityRule = ActivityTestRule(TestActivity::class.java)
+    private lateinit var testActivity: Activity
+
+    private lateinit var youTubePlayerView: YouTubePlayerView
+
+    private val clickTestViewId = R.id.youtube_player_view
+
+    @Before
```

Comment:
Group like this:

private lateinit var testActivity: Activity
private lateinit var youTubePlayerView: YouTubePlayerView

@Rule
@JvmField
val activityRule = ActivityTestRule(TestActivity::class.java)